### PR TITLE
Update 5.0.introduce-solution.md

### PR DESCRIPTION
**Pull request by foesa-yang**

### CODE REVIEW

File: docs-2.0/nebula-cloud/5.solution/5.1.supporting-application.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+|tools|OWNER|ROOT|USER|
+|:---|:---|:---|:---|
+|Dashboard|Y|Y|N|
+|Studio|N|Y|Y|
+|Explorer|N|Y|Y|
+
+## Dashboard
+
+Nebula Dashboard Enterprise(Dashboard for short) is a visualization tool that monitors and manages the status of machines and services in Nebula Graph clusters. Currently Dashboard in Cloud only supports viewing the [Cluster monitoring](../../nebula-dashboard-ent/4.cluster-operator/2.monitor.md) page.
```
Comment (abby-cyber) [inline docs-2.0/nebula-cloud/5.solution/5.1.supporting-application.md:15]:
```suggestion
Nebula Dashboard Enterprise (Dashboard for short) is a visualization tool that monitors and manages the status of machines and services in Nebula Graph clusters. Currently Dashboard in Cloud only supports [cluster monitoring](../../nebula-dashboard-ent/4.cluster-operator/2.monitor.md).
```

File: docs-2.0/nebula-cloud/5.solution/5.2.connection-configuration-and-use.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Private Link
+
+You can set up a Private Link in Connectivity to allow users to access the Nebula Graph database through a dedicated IP in the virtual network, without exposing your traffic to the public internet. For more information about private link, see [What is Azure Private Link?](https://docs.microsoft.com/en-us/azure/private-link/private-link-overview)
```
Comment (abby-cyber) [inline docs-2.0/nebula-cloud/5.solution/5.2.connection-configuration-and-use.md:3]:
```suggestion
You can create a Private Link endpoint in Connectivity to allow users to access Nebula Graph databases through a dedicated IP in a virtual network, without exposing your traffic to the public internet. For more information about Private Link, see [What is Azure Private Link?](https://docs.microsoft.com/en-us/azure/private-link/private-link-overview).
```

File: docs-2.0/nebula-cloud/5.solution/5.2.connection-configuration-and-use.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
+# Private Link
+
+You can create a Private Link endpoint in **Connectivity** to allow users to access Nebula Graph databases through a private IP in a virtual network, without exposing your traffic to the public internet. For more information about Private Link, see [What is Azure Private Link?](https://docs.microsoft.com/en-us/azure/private-link/private-link-overview).
+
+## Configure Private Link
+
+1. Enter your subscription ID, click **Create**. The creation time takes about 2 minutes.
+
+   !!! note
```
Comment (randomJoe211) [inline docs-2.0/nebula-cloud/5.solution/5.2.connection-configuration-and-use.md:9]: Make sure the indentation is correct in this topic.

File: docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+|Roles in Cloud|Roles in Nebula Graph|
+|:---|:---|
+|OWNER|-|
+|ROOT|ROOT|
+|USER|ADMIN/DBA/GUEST/USER|
+
+## Root Management
+
+Only users with **OWNER** authority can manage **ROOT** users.
```
Comment (randomJoe211) [inline docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md:13]: **OWNER** is a role. 
And if you use the bold format for roles, use it all the time. The next sentence does not have it.

File: docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+On the Root Management page, OWNER can reset the ROOT users.
```
Comment (randomJoe211) [inline docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md:15]: Bold for Root Management?

File: docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+Click `Reset`, enter the email address of the ROOT user to be updated, and click `Send Email` to send the email. After the designated user receives the confirmation email, click Confirm.
```
Comment (randomJoe211) [inline docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md:17]: "After the designated user receives the confirmation email, click Confirm."

1. Clarify who is the subject.
2. Use the bold format for UI keywords.

File: docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+## User Management
+
+Only users with **ROOT** authority can manage **USER** users.
```
Comment (randomJoe211) [inline docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md:21]: Same as preceding.

File: docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+
+On the USER management page, ROOT can specify the user's authority to the graph space as `ADMIN`, `DBA`, `GUEST` ,or `USER`.
```
Comment (randomJoe211) [inline docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md:23]: Is this page `User Management` or `USER management`? And bold the keywords.

The ROOT user can grant roles in graph spaces to other users. Available roles are ADMIN, DBA, GUEST, and USER.

File: docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+
+Select the graph space, click `Add User`, enter the specified user email and select the management authority of the user. Click `Send Email` to send the email. After the designated user receives the confirmation email, click Confirm.
```
Comment (randomJoe211) [inline docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md:25]: These are a list of steps. Why not separate them?

File: docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
+# Roles and authority management
+
+The role on the Cloud is different from that of the Nebula Graph. For more information, see [Roles in Cloud Solution](../4.user-role-description.md).
```
Comment (abby-cyber) [inline docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md:3]:
```suggestion
Nebula Graph Cloud roles are different from roles in Nebula Graph. For more information, see [Roles in Cloud Solution](../4.user-role-description.md).
```

File: docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md (hunk 5 above)
Comment (abby-cyber) [inline docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md:15]:
```suggestion
On the Root Management page, OWNER can reset ROOT users.
```

File: docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md (hunk 8 above)
Comment (abby-cyber) [inline docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md:23]:
```suggestion
On the USER management page, ROOT can grand users access to a graph space as `ADMIN`, `DBA`, `GUEST` ,or `USER`.
```

Comment (abby-cyber) [reply docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md:23]: plz confirm it is all the users or just some specified users that  ROOT can grand access to graph spaces?

File: docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md (hunk 5 above)
Comment (foesa-yang) [reply docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md:15]: This is the interface word

**Review by randomJoe211 — APPROVED** [review]

**Review by abby-cyber — APPROVED** [review]

**Pull request merged**